pkg/config/legacy/converter.go

```diff
@@ -437,3 +435,18 @@ func buildHistogramPercentiles(agentConfig Config) []string {
 
 	return histogramPercentile
 }
+
+func splitString(s string, sep rune) ([]string, error) {
+	r := csv.NewReader(strings.NewReader(s))
+	r.TrimLeadingSpace = true
+	r.LazyQuotes = true
+	r.Comma = sep
+
+	record, err := r.Read()
```

Comment:
could you just return r.Read() ?

Reply:
Indeed, silly me. I copy pasted this from the `trace-agent` without giving it a thought.